src/org/httpkit/server.clj

```diff
@@ -7,6 +7,8 @@
            [java.nio.channels ServerSocketChannel]
            java.security.MessageDigest))
 
+(set! *warn-on-reflection* true)
```

Comment:
I added a lot of type hints to avoid reflection and also enabled `*warn-on-reflection* to catch those better.

Reply:
Is the location of this intentional?
Won't this override the user's intended `*warn-on-reflection*` on ns eval?

Reply:
It will always enable reflection warnings in the server namespace, which is what we want to always prevent, right?
It doesn't have an effect on other namespaces.

Reply:
Sorry - I might be missing something obvious here, but `clojure.core/*warn-on-reflection*` is a global var - how would setting it not affect other namespaces?

Reply:
Interesting, I see I'm mistaken! The [`set!` doc](https://clojure.org/reference/vars#set) indeed clarifies that it's only a thread-local effect.

I've somehow never encountered this before, that's actually very handy to know!

Thanks for pointing this out :-)

Reply:
You're welcome :)